tests/core/data/test_data_manager.py

```diff
@@ -451,3 +452,57 @@ def test_delete_does_clean_generated_pickle_files(self, pickle_file_path):
 
         _DataManager._delete_all()
         assert not file_exists(generated_pickle_dn_3.path)
+
+    def test_create_dn_when_config_storage_type_none(self):
+
+        file_config = NamedTemporaryFile(
+            # """
+            # [TAIPY]
+            # [DATA_NODE.input]
+            # scope = "SCENARIO:SCOPE"
+            # default_data = "21:int"
+            # [DATA_NODE.output]
+            # scope = "SCENARIO:SCOPE"
+            # """
+            # )
+            """
+            [TAIPY]
+
+            [DATA_NODE.input]
+            scope = "SCENARIO:SCOPE"
+            default_data = "21:int"
+
+            [DATA_NODE.output]
+            scope = "SCENARIO:SCOPE"
+
+            [TASK.double]
+            inputs = [ "input:SECTION",]
+            function = "math.sqrt:function"
+            outputs = [ "output:SECTION",]
+            skippable = "False:bool"
+
+            [PIPELINE.my_pipeline]
+            tasks = [ "double:SECTION",]
+
+            [SCENARIO.my_scenario]
+            pipelines = [ "my_pipeline:SECTION",]
+
+            [SCENARIO.my_scenario.comparators]
+            """
+        )
+
+        Config.load(file_config.filename)
+
+        from src.taipy import core as tp
+
+        scenario = tp.create_scenario(Config.scenarios["my_scenario"])
+
+        assert isinstance(scenario.input, PickleDataNode)
+        assert isinstance(scenario.output, PickleDataNode)
+
+        # Config._register_default(DataNodeConfig('default_id', 'csv'))
```

Comment:
@jrobinAV 
I want to add a test where we change the default storage type and the data manager should change the default class used for dn as well. But _register_default doesn't seem to do the work. How can I change the default storage type for dn in the config?